.github/workflows/ci.yml

```diff
@@ -1,18 +1,11 @@
-name: Continuous integration
-
 on: [pull_request, push]
-
 jobs:
-  check_pr:
+  # The main CI workflow; installs dependencies, runs eslint, prettier, and jest.
+  main:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-
-    - name: "npm ci"
-      run: npm ci
```

Comment:
I tried to make things a little more compact by removing the `name` fields here since they were just echoing the command.